backend/siarnaq/bracket/challonge.py

```diff
@@ -59,6 +59,8 @@ def create_tournament(tournament: Tournament, *, is_private: bool):
                 "tournament_type": challonge_type,
                 "private": is_private,
                 "url": challonge_id,
+                "game_name": "Battlecode",
```

Comment:
These changes just set up our tournaments to support replay url attachments in the future, correct?

Reply:
Also, will we need to merge #723 before merging this PR? b/c I see changes from camelcase to snakecase in that PR, but i'm not sure if that's a challonge v2 -> v2.1 change or challonge v1 -> v2  change

Reply:
Yup (and also they do more things)
 updated PR description for more context

Reply:
ah good catch, idk if "game_name" is v2 or v2.1
I'll play it safe